README.md

```diff
@@ -44,11 +42,13 @@ If your game uses Rakugo, tell us on [Discord](https://discord.gg/K9gvjdg).
 
 To install Rakugo plugin, download it as a [ZIP archive](). Extract the `addons/Rakugo` folder into your project folder. Then, enable the plugin in project settings and restart Godot-Engine.
 
-If you want to know more about installing plugins you can read the [official documentation page](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/installing_plugins.html).
+If you want to know more about installing plugins you can read the [godot official documentation page](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/installing_plugins.html).
 
 ## Get Started
```

Comment:
@theludovyc Some can found this first instead of Rakugo website/itch.io, so you should mention that there are also Kits to start from here: https://rakugoteam.github.io/download/

Reply:
I don't understand, why do you want to mention kits here ? ^^'

Reply:
I think someone can found this repo instead of Rakugo website/itch.io, 
so he/she can think this only way there is to interact with Rakugo.

Reply:
If a user find this repo first. In some days we have a release. So we can add a link to download it in readme. I think it's better if https://rakugoteam.github.io/download/ refer to this page ?

Reply:
For now, you are right, but in feature there be more Kits then just VNKit, 
and I made Rakugo page in such way that anyone can propose his/her kit thought PR to Rakugo website repo.